src/backend/gl/src/command.rs

```diff
@@ -108,6 +108,7 @@ pub enum Command {
     SetPatchSize(i32),
     BindProgram(<GlContext as glow::Context>::Program),
     BindBlendSlot(ColorSlot, pso::ColorBlendDesc),
```

Comment:
Did you want both BindAllBlendSlots *and* BindBlendSlot changed to SetXXX?

Reply:
yes, thank you!